app/routes/upload.step3.tsx

```diff
@@ -0,0 +1,5 @@
+const step3 = () => {
+  return <div>ImproveResume</div>
+}
```

Comment:
**nitpick**: Consider adding more descriptive content or instructions for the user on this step.